.env

```diff
@@ -74,3 +74,7 @@ ROUTEMASTER_MONITOR_INTERVAL=10
 ROUTEMASTER_SCALING_THRESHOLD=100
 ROUTEMASTER_SCALING_DEADLINE=1000
 
+# maximum number of bytes in an event's optional data payload (when messagepack'ed)
+# DO NOT INCREASE THIS - using the event bus as a data bus is a strong smell of
+# an incompatible architecture, and will cause major bus scalability issues.
+ROUTEMASTER_MAX_EVENT_DATA=64
```

Comment:
"just" the data fields (the last three) encodes to 50-60 byte depending on significant digits.

Would we want to carry the `_links` section on the bus for these payloads?
Do we need a `created_at`, given that this is for immutable data and the bus already carries a timestamp?